packages/api/src/payments/payrexxPaymentProvider.ts

```diff
@@ -125,7 +125,15 @@ export class PayrexxPaymentProvider extends BasePaymentProvider {
         method: 'GET'
       }
     )
-    if (res.status !== 200) throw new Error(`Payrexx response is NOK with status ${res.status}`)
+    if (res.status !== 200) {
+      logger('payrexxPaymentProvider').error(
+        res,
+        'Payrexx response for intent %s is NOK with status %s',
+        intentID,
```

Comment:
What do you think about logging object/hash instead of flat values. 
```
logger('payrexxPaymentProvider').error({
  res,
  message: 'Payrexx response for intent %s is NOK with status %s',
  intentID,
  status: res.status
})
```
I did that in previous project and usually what logging viewers do with that is they present message plus you can expand json payload when needed.

Reply:
Still not sure if that's a place for this change - maybe a general rework of logging in another PR. @roosnic1 please share your thoughts.